.github/workflows/pr-validations.js

```diff
@@ -0,0 +1,132 @@
+const path = require("path")
```

Comment:
Perhaps this makes more sense under `tools/`

Reply:
see above